core/logger/default.go

```diff
@@ -52,125 +53,126 @@ func SetLogger(newLogger Logger) {
 		}(Default)
 	}
 	Default = newLogger
+	skipDefault = Default.withCallerSkip(1)
```

Comment:
These helper funcs need to skip another level so they don't reference lines from this file.